.travis.yml

```diff
@@ -11,11 +11,4 @@ env:
   # TODO: add test env (where DEBUG=False, secret key unencrypted)
   - HKNWEB_MODE=dev
 install:
-  # Dependencies
-  - pip install pipenv
-  - pipenv install
-  # Dev dependencies
-  - pipenv install --dev
 script:
-  - pipenv run python ./manage.py migrate
-  - pipenv run python ./manage.py check
```

Comment:
I think you'll still need something here? Otherwise I don't think it'll know how to do any setup or run any tests

Reply:
Not sure exactly what to put, I'll push something and see if it suffices lol

Reply:
Looks like Travis passes!